src/lumigo_opentelemetry/__init__.py

```diff
@@ -184,7 +210,35 @@ def init() -> Dict[str, Any]:
 
     trace.set_tracer_provider(tracer_provider)
 
-    return {"tracer_provider": tracer_provider}
+    if logging_enabled:
```

Comment:
you already have the same `IF` above, so I wonder why not merge these code blocks?

Reply:
The section above is nested within a check that there's a token present, so I kept it separate intentionally - so we will be able to dump logs to file regardless of the token. Do you think it's better to force having a token for the case of dumping logs to a file?